.clj-kondo/config.edn

```diff
@@ -395,7 +395,12 @@
                                      ;; TODO -- we can't make this part of [[metabase.server.core]] YET because it
                                      ;; causes circular dependencies. Maybe in the future once we tease apart more of
                                      ;; the modules tangle we can.
-                                     metabase.server.protocols}
+                                     metabase.server.protocols
+                                     ;;
+                                     ;; TODO -- I really don't like having YET ANOTHER allowed external namespace for
+                                     ;; this module but some things need to import it so
+                                     ;; `metabase.server.streaming_response.StreamingReponse` comes into existence...
+                                     metabase.server.streaming-response}
```

Comment:
same

Reply:
Overall the surface area of the project as a whole is still less because the old `async` module had several namespaces -- `server` needed to dig into `metabase.async.streaming-response.thread-pool`, now it's entirely internal. But this is still less than ideal. I think once we get further along we can reduce the surface area of some of these modules. Right now there are too many circular deps to fix it just yet